client/src/templates/Guide/components/calculators/CodingBootcampCostCalculator/CodingBootcampCostCalculator.js

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+
+import ArticleLayout from '../../ArticleLayout';
+
+import CostCalculatorChart from './CostCalculatorChart';
+import './CodingBootcampCostCalculator.css';
+
+const propTypes = {
+  data: PropTypes.object
+};
+
+class CostCalculator extends React.Component {
+  state = {
+    cities: [],
+    incomes: [],
+    city: null,
+    cityLabel: '_______',
+    lastYearsIncome: null,
+    lastYearsIncomeLabel: '_______',
+    bootcamps: null
+  };
+
+  initComponent() {
+    fetch('/json/bootcamps.json')
```

Comment:
Let's place `bootcamps.json` in to `md` file, like this:
```html
<script type="application/json" id="bootcamps">
[
  {
    "name": "Hack Reactor",
    "cost": "17780",
    "housing": "500",
    "weeks": "12",
    "finance": true,
    "cities": ["new-york-city", "san-francisco"]
  },
...
</script>
```
and replace link to `bootcamps.json`:
```html
<a
  href='javascript:void(0)'
  download='bootcamps.json'
  id='bootcamps-data-link'
>
  Save Data Source JSON
</a>
```
then in component, we can do somethig like this:
```js
    const bootcamps = JSON.parse(
      document.getElementById('bootcamps').innerHTML
    );
    document
      .getElementById('bootcamps-data-link')
      .setAttribute(
        'href',
        `data:text/plain;charset=utf-8,${encodeURIComponent(
          JSON.stringify(bootcamps, null, 2)
        )}`
      );
```